drafts/all-we-need-is-structure.md

```diff
@@ -0,0 +1,157 @@
+---
+title: All we need is Structure
```

Comment:
Maybe I would add a prefix or suffix to your series with part 1, 2, 3 so people know in which sequence to read them (also existing merged ones)